examples/eventhub_management.bal

```diff
@@ -68,9 +68,9 @@ public function main() {
         log:printError(listResult.message());
     }
     if (listResult is stream<azure_eventhub:EventHub>) {
-        _ = listResult.forEach(isolated function (azure_eventhub:EventHub eventHub) {
-                log:printInfo(eventHub.toString());
-            });
+        _ = listResult.forEach(isolated function(azure_eventhub:EventHub eventHub) {
+            log:printInfo(eventHub.toString());
+        });
```

Comment:
```suggestion
        _ = listResult.forEach(isolated function(azure_eventhub:EventHub eventHub) {
                log:printInfo(eventHub.toString());
            });
```

Reply:
@nipunayf this seems to be an issue in the Ballerina formatter. So better if we create an issue in the ballerina lang repo  for this